analyzer/pkg/rpm/rpm.go

```diff
@@ -84,7 +87,10 @@ func (a rpmPkgAnalyzer) parsePkgInfo(packageBytes []byte) ([]types.Package, []st
 		var srcName, srcVer, srcRel string
 		if pkg.SourceRpm != "(none)" && pkg.SourceRpm != "" {
 			// source epoch is not included in SOURCERPM
-			srcName, srcVer, srcRel = splitFileName(pkg.SourceRpm)
+			srcName, srcVer, srcRel, err = splitFileName(pkg.SourceRpm)
+			if err != nil {
+				log.Printf("Invalid Source RPM Found: %s", pkg.SourceRpm)
```

Comment:
Please use the logger here.
https://github.com/aquasecurity/fanal/blob/main/log/log.go

```suggestion
				log.Logger.Debugf("Invalid Source RPM Found: %s", pkg.SourceRpm)
```